blackbox_repository/conversion_scripts/recipes.py

```diff
@@ -1,10 +1,10 @@
-from blackbox_repository.conversion_scripts.scripts.nas201_import import generate_nas201
+from blackbox_repository.conversion_scripts.scripts.nasbench201_import import generate_nasbench201
 from blackbox_repository.conversion_scripts.scripts.fcnet_import import generate_fcnet
 from blackbox_repository.conversion_scripts.scripts.icml2020_import import generate_deepar, generate_xgboost
 
 generate_blackbox_recipe = {
     "icml-deepar": generate_deepar,
     "icml-xgboost": generate_xgboost,
-    "nas201": generate_nas201,
+    "nasbench201": generate_nasbench201,
```

Comment:
If we make this change, we should also update the example: https://github.com/awslabs/syne-tune/blob/main/examples/launch_simulated_benchmark.py#L86

Reply:
+1